.github/workflows/branch_deployments.yml

```diff
@@ -6,31 +6,51 @@ concurrency:
   # Cancel in-progress runs on same branch
   group: ${{ github.ref }}
   cancel-in-progress: true
-
 env:
   DAGSTER_CLOUD_URL: ${{ secrets.DAGSTER_CLOUD_URL }}
+  DAGSTER_CLOUD_API_TOKEN: ${{ secrets.DAGSTER_CLOUD_API_TOKEN }}
 
 jobs:
-  parse_workspace:
+  dagster_cloud_default_deploy:
+    name: Dagster Serverless Deploy
     runs-on: ubuntu-latest
     outputs:
       build_info: ${{ steps.parse-workspace.outputs.build_info }}
+    
     steps:
-      - uses: actions/checkout@v3
       - name: Parse cloud workspace
+        if: env.ENABLE_PYTHON_EXECUTABLE != 'true'
         id: parse-workspace
         uses: dagster-io/dagster-cloud-action/actions/utils/parse_workspace@v0.1
         with:
           dagster_cloud_file: dagster_cloud.yaml
 
-  dagster_cloud_build_push:
+      - name: Checkout
+        if: env.ENABLE_PYTHON_EXECUTABLE == 'true'
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.sha }}
+          path: project-repo
+          
+      - name: Build and deploy Python executable
+        if: env.ENABLE_PYTHON_EXECUTABLE == 'true'
+        uses: dagster-io/dagster-cloud-action/actions/build_deploy_python_executable@pex-testing
```

Comment:
note, we will probably have to keep this tag around forever (lest we break users who opt into this), since they will have cloned this into their own repo.

consider changing this to something like `v0.1` or something like that, that we feel comfortable moving?

Reply:
good catch. `v0.1` [already exists](https://github.com/dagster-io/dagster-cloud-action/tags) and I don't want to move it to HEAD where most of the newer pex changes will land. should i make this `v0.2`?

Reply:
ok, changed to `@pex-v0.1` so this tag is independent of the main releases for now.

Reply:
I think it's acceptable to also just do `v0.1.10` or something like that.  We should be moving the `v0.1` tag along with the minor versions, as long as there are no breaking changes.

If we want to make backwards incompatible changes, we could bump to `v0.2` and then all the existing repos would need to change the tag to take advantage of the newer stuff.